extensions/default/src/DicomWebDataSource/utils/StaticWadoClient.js

```diff
@@ -56,8 +65,17 @@ export default class StaticWadoClient extends api.DICOMwebClient {
     const { queryParams } = options;
     if (!queryParams) return searchResult;
     const filtered = searchResult.filter(study => {
-      for (const key of Object.keys(StaticWadoClient.studyFilterKeys)) {
-        if (!this.filterItem(key, queryParams, study)) return false;
+      for (const key of Object.keys(StaticWadoClient.seriesFilterKeys)) {
```

Comment:
For Series, shouldn't we send the combination of the `studyFilterKeys` and `seriesFilterKeys` ?

Reply:
if `searchForSeries()` is called after `searchForStudies()` , then it would be fine.